services/r2ctl/main/main.go

```diff
@@ -126,17 +117,17 @@ func main() {
 		logger.Fatalf("No valid port configured. Can't start.")
 	}
 
-	scope, closer, err := cfg.Metrics.NewRootScope()
-	if err != nil {
-		logger.Fatalf("error creating metrics root scope: %v", err)
-	}
-	defer closer.Close()
+	// scope, closer, err := cfg.Metrics.NewRootScope()
```

Comment:
assuming this is for testing and you'll undo before merge, yea?

Reply:
Yep.

Reply:
Cool